### PR TITLE
Insert testplatform v15.9.0-preview-20180924-03

### PR DESCRIPTION
**Pull request by smadala**
- Release notes: https://github.com/Microsoft/vstest-docs/pull/154
- VS insertion PR: https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/142944?_a=overview

### CODE REVIEW

**Comment by smadala** [comment]
@livarcocc Is `release/2.1.5xx` right branch for VS 15.9?

**Comment by livarcocc** [comment]
@smadala yes.

File: test/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestfromCsproj.cs [hunk 1]
```diff
@@ -72,7 +72,7 @@ public void ItDoesNotImplicitlyRestoreAProjectWhenTestingWithTheNoRestoreOption(
 
             new DotnetTestCommand()
                 .WithWorkingDirectory(testProjectDirectory)
-                .ExecuteWithCapturedOutput($"{TestBase.ConsoleLoggerOutputNormal} --no-restore")
+                .ExecuteWithCapturedOutput($"{TestBase.ConsoleLoggerOutputNormal} --no-restore /p:IsTestProject=true")
```
Comment (livarcocc) [inline test/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestfromCsproj.cs]: Why was this change needed?

Comment (smadala) [reply test/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestfromCsproj.cs]: Behavior change due to https://github.com/Microsoft/vstest/pull/1745

`IsTestProject` property added by `Microsoft.NET.Test.sdk` package. As the test trying to execute tests without restoring we are not getting expected exit code/error message. 

I think we should add info message when skipping a project if there is no `IsTestProject` property then update this test to asset on same info message. If this behavior sounds good we will take this change separately(next insertion).

Comment (smadala) [reply test/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestfromCsproj.cs]: @livarcocc I agree with you. I have created https://github.com/Microsoft/vstest/issues/1775 to fix this test in next insertion. Can we merge this PR?

Comment (livarcocc) [reply test/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestfromCsproj.cs]: I would rather not, as this is a regression in behavior.

Comment (smadala) [reply test/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestfromCsproj.cs]: @livarcocc We can't fail the `dotnet test` command if there is no `IsTestProject` prop. The behavior change is intended we should remove this test.
